app/Jobs/ExportPostsJob.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Ushahidi\App\Jobs;
+
+use Exception;
+use Ushahidi\Core\Usecase\Export\Job\PostCount;
+use Ushahidi\Core\Entity\ExportJob;
+use Ushahidi\Core\Entity\ExportJobRepository;
+use Illuminate\Support\Facades\Log;
+
+class ExportPostsJob extends Job
+{
+    protected $batchSize = 200;
```

Comment:
could this live in a config, but have 200 as a default?

Reply:
It could. I'll see if I have time since I think it was hard coded before